test/tests/inittest.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+# This have to be first test for cockpit
+#  install build dependencies
+#  download new version of cockpit from git 
+#  compile it
+# it is basic test compilation must PASS
+
+set -x
+
+BUILD="build1"
+PACKAGE="cockpit"
+SOURCE="https://github.com/$PACKAGE-project/$PACKAGE.git"
+
+#
+yum-builddep -y $PACKAGE
+yum-builddep -y $PACKAGE
```

Comment:
It is tricky part :-) 
I've met situation that first installation failed, some strange reason, but second one passed well. So second one is there only to ensure that it installed.
Probably it was caused, that at first run, yum-builddep has not been ever running, so something there (It is bug, but I have to somehow deal with that)